src/gl_context.rs

```diff
@@ -29,7 +29,13 @@ impl<Native> GLContext<Native>
     where Native: NativeGLContextMethods
 {
     pub fn create(shared_with: Option<&Native::Handle>) -> Result<GLContext<Native>, &'static str> {
-        let native_context = try!(Native::create_shared(shared_with));
+        Self::create_shared(shared_with, None)
+    }
+
+    pub fn create_shared(shared_with: Option<&Native::Handle>,
```

Comment:
Maybe `shared_with_dispatcher` (or `create_shared_with_dispatcher`)? `create_shared` is a bit confusing, since the normal `create` function also creates a shared context.